torch/_overrides.py

```diff
@@ -825,25 +1049,37 @@ def get_overridable_functions():
         (torch, torch.__all__ + dir(torch._C._VariableFunctions)),
         (torch.functional, torch.functional.__all__),
         (torch.nn.functional, dir(torch.nn.functional)),
+        (torch.Tensor, dir(torch.Tensor))
     ]
     for namespace, ns_funcs in tested_namespaces:
         for func_name in ns_funcs:
             # ignore private functions or functions that are deleted in torch.__init__
-            if func_name.startswith('_') or func_name == 'unique_dim':
+            if namespace is not torch.Tensor and func_name.startswith('_') or func_name in {'unique_dim', '__weakref__'}:
```

Comment:
Just a nit, you could avoid checking `namespace is not torch.Tensor` so many times with something like this:
```
if namespace is not torch.Tensor:
    if func_name.startswith('_'): 
        continue
    elif func_name.endswith('_'):
        continue
    elif not func_name.islower():
        continue
else:
    func = getattr(namespace, func_name)
    if getattr(object, func_name, None) == func:
        continue
```